geode-assembly/src/integrationTest/java/org/apache/geode/BundledJarsJUnitTest.java

```diff
@@ -67,7 +67,7 @@ public void verifyBundledJarsHaveNotChanged() throws IOException {
         sortedJars.entrySet().stream().map(entry -> removeVersion(entry.getKey()));
     Set<String> bundledJarNames = new TreeSet<>(lines.collect(Collectors.toSet()));
 
-    Files.write(Paths.get("bundled_jars.txt"), bundledJarNames);
+    Files.write(Paths.get("..", "bundled_jars.txt"), bundledJarNames);
```

Comment:
This change looks good, however, BundledJarsJUnitTest is not the only test that has this problem; AssemblyContentsIntegrationTest and GeodeDependencyJarIntegrationTest also need essentially the same change.

Reply:
Can you think of way to identify the tests that have this problem?

Reply:
I did a scan of the integration test output to spot this kind of file. I found AssemblyContentsIntegrationTest and GeodeDependencyJarIntegrationTest. I will fix those. If you know of others, let me know.

Reply:
The three mentioned tests are the only ones I'm aware of that write files that need to be accessed after the test runs. I searched for all other occurrences of "Files.write" and none seemed important. I'm not sure there is a better way to identify which files are important vs random temp files.